golang/sqlboiler/graph/templates/singleton/helper.tpl

```diff
@@ -0,0 +1,54 @@
+const (
+  // defaultPageSize is used in AllXxx query methods `args.PageSize`
+  defaultPageSize = 25
+)
+
+// QueryModPagination returns SQLBoiler QueryMods for pagination 
+func QueryModPagination(pageNum, pageSize *int32) []qm.QueryMod { 
+  // Page size
+  limit := defaultPageSize // Default page size
+	if pageSize != nil {
+		limit = int(*pageSize)
+	}
+
+  // Page number
+  if pageNum == nil || pageSize == nil {
+    return []qm.QueryMod{qm.Limit(limit)}
+  }
+
+  offset := (int(*pageNum) * int(*pageSize))
+  return []qm.QueryMod{qm.Limit(limit), qm.Offset(offset)}
+}
+
+// QueryModsSearch returns a list of QueryMod based on the struct values
+func QueryModsSearch(input interface{}) []qm.QueryMod {
```

Comment:
this deprecates the entire https://github.com/choonkeat/hellocrud/blob/6261f7c4b31bd750f85271d084978518a8461a7c/golang/sqlboiler/graph/templates/03_search_args.tpl correct?

Reply:
Err.. Sorry, forgot to delete as well, like constants.tpl. 🙏🏻 Will be aware of delete/rename diffs next time. (Working on my own repo and copying `templates` folder into hellocrud) 😅

Reply:
i was about to ask you how you're using this repo. i see now ;-)